ui/nuxeo-layout.js

```diff
@@ -125,7 +122,17 @@ import './nuxeo-error.js';
           valid = (el.validate ? el.validate() : el.checkValidity()) && valid;
         }
       }
-      return valid;
+      if (!valid) {
+        return false;
+      }
+      if (this.element && typeof this.element.validate === 'function') {
```

Comment:
Why did we move this part to the end? It changes the current behavior.

Reply:
Yes. Because it does make more sense to me to evaluate native form input validation beforehand potential async layout validation. The layout's validation can now be async which means it could take time to return while form inputs validation is immediate.

Reply:
Yes, it makes sense to me. 
But, if the validation is not async and the element has a sync validation it may never be executed, correct? Are ok with this behavioural change?